src/framework/analyses.ml

```diff
@@ -143,23 +143,13 @@ struct
 end
 
 
-(* Experiment to reduce the number of arguments on transfer functions and allow
-   sub-analyses. The list sub contains the current local states of analyses in
-   the same order as written in the dependencies list (in MCP).
-
-   The foreign states when calling special_fn or enter are joined if the foreign
-   analysis tries to be path-sensitive in these functions. First try to only
-   depend on simple analyses.
-
-   It is not clear if we need pre-states, post-states or both on foreign analyses.
-*)
```

Comment:
This comment was hilariously outdated. There should still be some documentation comment `(** ... *)` saying it's a "manager" and perhaps something more.